CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+If you have a plugin that you want to appear on this list, please open a PR.  
+We will review the plugin and give an opinion about it. If the opinion is
+favorable then we will update your PR with our comment and merge it.
+
+Your plugin MUST be high quality. We accept only plugins with a code quality
+of 9.80 out of 10 and above.
+
+Also it has to be:
```

Comment:
We add a word about Tests ?
No need for 100% test coverage, but plugin have the minimum testing that covers major functionality?